packages/core/src/agent/didcomm/versions/v1/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { Key } from '../../../../crypto'
+import type { DecryptedMessageContext, EncryptedMessage, SignedMessage, MessageType } from '../../types'
+import type { AgentContext } from './../../../context'
+import type { DIDCommV1Message } from './DIDCommV1Message'
+
+export { DIDCommV1Message } from './DIDCommV1Message'
+export { DIDCommV1BaseMessage, DIDComV1BaseMessageConstructor } from './DIDCommV1BaseMessage'
+
+export interface PackMessageParams {
+  recipientKeys: Key[]
+  routingKeys: Key[]
+  senderKey: Key | null
```

Comment:
This seems to work with keys, while the didcomm v2 service works with dids. Can we get the on the same level somewhow? I.e. DIDComm v1 also get's dids as input or vice versa?

Or do you think it's better to have this differation between the two ?